pymilo/transporters/transporter.py

```diff
@@ -56,6 +55,26 @@ def deserialize(self, data, key, model_type):
         :return: pymilo deserialized output of data[key]
         """
 
+    @abstractmethod
+    def bypass(self, content):
+        """
+        Determine whether to bypass transporting on this content or not.
+
+        :param content: either a ML model object's internal data dictionary or an object associated with the json string of a pymilo serialized ML model.
```

Comment:
Isn't it better?
```
        :param content: data dictionary or json object of a model
```

Reply:
I think it is more comprehensive, are you sure we don't need to add extra information?